### DISCUSSION TITLE
Http task for AWS fails in conductor with authorization

### DISCUSSION THREAD

**Discussion by mummoo** [Netflix/conductor#2276] [Q&A]
I am seeing an issue where the https request to AWS service (with authorization) returns successfully when run directly (using python requests and nodejs axios), but it fails when being run in conductor.
```

"http_request": {
               "headers": {
                  "Content-Type": "application/x-amz-json-1.0", 
                  "host": "lambda.us-west-2.amazonaws.com",
                  "X-Amz-Date": "20210526T232100Z",
                  "Authorization": "AWS4-HMAC-SHA256 Credential=acc-id/20210526/us-west-2/lambda/aws4_request, SignedHeaders=content-type;host;x-amz-date, Signature=aabbbcccccdddddddddd"
               },
               "method": "POST",
               "body": "{}",
               "uri": "https://lambda.us-west-2.amazonaws.com/2015-03-31/functions/function_name/invocations",
               "connectionTimeOut": 6000
            }
```

When I run this as a simple post request with the url, body, and headers, it works. 
However, same request fails in conductor.

_Failed to invoke HTTP task due to: java.lang.Exception: 403 Forbidden: [{"message":"The request signature we calculated does not match the signature you provided. Check your AWS Secret Access Key and signing method. Consult the service documentation for details."} ]_

Any idea what may be wrong? does it support header properties defined as above?

**Comment by kishorebanala** [comment]
Http task passes through the headers as is (https://github.com/Netflix/conductor/blob/main/contribs/src/main/java/com/netflix/conductor/contribs/tasks/http/HttpTask.java#L151). Do you know how the request signature is calculated, and if it allows requests from Conductor?
